DataProcessing/utils.py

```diff
@@ -143,8 +163,43 @@ def viz_data_on_video(input_vid, output_path, pre_labeled_pkl_path=None,path_to_
     mmcv.frames2video(temp_path, output_path, fps=fps, fourcc='mp4v', filename_tmpl='{:03d}.png')
     temp_dir.cleanup()
 
+
+def create_tracklet_hist(pre_labeled_pkl_path):
```

Comment:
Let's merge this code already with this PR so we don't need the other one.

Reply:
since we are removing the CROP obj this will be deprecated. I suggest to comment it out for now, if we like to adapt it to the DB we can do so later